features/ubuntu_upgrade.feature

```diff
@@ -13,8 +14,8 @@ Feature: Upgrade between releases when uaclient is attached
         AllowThirdParty=yes
         """
         And I run `sed -i 's/Prompt=lts/Prompt=normal/' /etc/update-manager/release-upgrades` with sudo
-        Then I verify that running `do-release-upgrade <devel_release> --frontend DistUpgradeViewNonInteractive` `with sudo` exits `0`
-        When I reboot the `<release>` machine
+        And I run `do-release-upgrade <devel_release> --frontend DistUpgradeViewNonInteractive` `with sudo` and stdin `y\n`
```

Comment:
It looks like this is supposed to use the `stdin` var from the examples?

```suggestion
        And I run `do-release-upgrade <devel_release> --frontend DistUpgradeViewNonInteractive` `with sudo` and stdin `<stdin>`
```

Reply:
Or should we leave `y/n` hardcoded here and remove the `stdin` column in the examples?